docs/data/tree-view/simple-tree-view/focus/focus.md

```diff
@@ -0,0 +1,16 @@
+---
+productId: x-tree-view
+title: Simple Tree View - Focus
+components: SimpleTreeView, TreeItem
+packageName: '@mui/x-tree-view'
+githubLabel: 'component: tree view'
+waiAria: https://www.w3.org/WAI/ARIA/apg/patterns/treeview/
+---
+
+# Simple Tree View - Focus
+
+<p class="description">Learn how to focus Tree View items.</p>
+
+## Focus a specific node
+
+{{"demo": "FocusedSimpleTreeView.js"}}
```

Comment:
How should it behave when the parent of the node is collapsed?

I propose not to expand magically any node, but instead to give the tools to the users to expand the ancestors.
Either with an high level `expandNodeAncestors` method that expands the parent, grand parent etc..., or to make public `isNodeExpanded` + `toggleNodeExpansion` + `getNode` (the high level method is probably easier to reason with here).

In either case, I think it's worth adding a `info` note saying that we only support focusing nodes whom parents are expanded AND make sure that the behavior is not totally broken when a user calls `focusNode` with a collapsed node.

Reply:
Just to be sure, this part should IMHO be done in this PR

> I think it's worth adding a info note saying that we only support focusing nodes whom parents are expanded AND make sure that the behavior is not totally broken when a user calls focusNode with a collapsed node.